pkg/landscape/app/graph-store.hoon

```diff
@@ -857,8 +857,8 @@
       --
     ::
         [%depth-first @ @ ~]
-      =/  count=(unit atom)  (rush i.t.t.t.t.path dem:ag)
-      =/  start=(unit atom)  (rush i.t.t.t.t.t.path dem:ag)
+      =/  count=(unit atom)  (rush i.t.t.t.t.t.path dem:ag)
+      =/  start=(unit atom)  (rush i.t.t.t.t.t.t.path dem:ag)
```

Comment:
This isn't actionable right now, but it's unfortunate that this interface doesn't/can't use `(slaw %ud ...)`, which is jetted.